modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -449,18 +450,14 @@ class ApiClient(object):
                 new_params.append((k, v))
         return new_params
 
-    def prepare_post_parameters(self, post_params=None, files=None):
+    def files_parameters(self, files=None):
         """Builds form parameters.
```

Comment:
@rienafairefr the change looks good to me. One minor suggestion. Shall we update the method description above as well?

Reply:
Let's go with the current description. We can update it later with another PR if needed.